### ISSUE TITLE
Where does the example docking menu go on a resize to mobile?

### ISSUE THREAD

**Issue by mgalena** [dhg/Skeleton#219]
Maybe I'm missing something, but with the example page, when I resize it in a desktop browser window, or when I open it on my iPhone... the menu seems to disappear altogether. Can't find it anywhere to be seen.

It seems like maybe there's a missing icon to make a menu appear... or something like that.

To be clear, this observing the main page on http://getskeleton.com/

Is this a bug?

**Comment by dhg** [comment]
Not a bug, but rather a design decision that might need revisiting. I knew some type of nav was required to jump around the page, but later added other critical links offsite. Originally I figured it wasn't necessary on mobile, but now it definitely needs to be added.

**Comment by mgalena** [comment]
If you consider it a navigation menu that links you to other pages (as is today's norm), and not just content within the same page, then yes... maintaining access to it via a chicklet menu on a mobile interface would be preferred. Even if it is only accessing a part of the same page, if that page is remarkably long on a mobile device, quick access to the menu to jump to any <a> anchor point within the document is also a good idea, to prevent constant finger scrolling. 

Can we hope for an updated example any time soon?

Thank you!

**Comment by aquatix** [comment]
I agree with needing a navigation on mobile, especially to other pages, but in the same page too, like @matthewgalena notes.

**Comment by r3id** [comment]
I was just about to post a request for mobile navigation. Is there any progress on this?

**Comment by jamesa** [comment]
If @dhg decides a way he wants it done I'd be happy to PR.

**Comment by dcolumbus** [comment]
I'd love to see an example for the docking nav you created as well as a responsive menu.

**Comment by yedderson** [comment]
getskeleton.com main page freezes when scrolling on mobile (Chrome/Android) I guess it's because the page got too big, that's a terribly bad thing for someone considering using Skeleton.

**Comment by michaelwayman** [comment]
@yedderson it works fine on my device, GS IV. 
@matthewgalena agreed

**Comment by COLABORATI** [comment]
Would be nice to see growing a menu component for skeleton.

@michaelwayman - which value has a "works ok on my device" comment?

**Comment by itsdarrylnorris** [comment]
I build a Drupal theme (https://www.drupal.org/project/light_skeleton) base on this project and I had to build a menu mobile menu in order to release an MVP theme. You can see a demo of this menu at http://lightskeleton.com/

If you wish to use the css and js for this menu:
Here is the CSS - https://gist.github.com/darol100/dc7e2466f25cd54fad0d
Here is the JS - https://gist.github.com/darol100/113b70533aa5cb358e1a

In order to implement this menu you probably will need to change the html a little bit.

However, I'm not a fan of that menu so I have plans on rebuild that menu for a something more modern. I have open an issue for that at https://www.drupal.org/node/2576703. But even thought I want fix it for my Drupal theme, I think this should be fix at the skeleton project first in order to help improve this project and both projects get a benefit.

My question to the maintainer is going to be.... Would you be willing to add a JS file to your project in order to get a modern simple mobile nav ? @devanandb, is going to be helping me on creating this menu and this is very important for us to know if you are ok adding this JS as part of Skeleton.

**Comment by ganeshkbhat** [comment]
Nice theme but the mobile navigation is surely missing something thats needed. Any status? A very simple mobile compatible menu skeleton should suffice

**Comment by inkplug** [comment]
I'm working with Skeleton and noticed this as well. Other than the lack of mobile menu, loving it so far!

**Comment by rotaercz** [comment]
Would like to see a responsive navigation as well.

**Comment by jondoesntgit** [comment]
+1

**Comment by ntabernacle** [comment]
+1

**Comment by sociomarker** [comment]
Should have provision for multi level menus..

**Comment by CaffeineLab** [comment]
A starting point is to move everything from @media (min-width: 750px)  to the top section of custom.css.  I moved it under .example-screenshot.coming-soon{} and it works with one important caveat - it doesn't wrap to a  second line very well.  I didn't spend time trying to solve it because I only had three items in my menu so it didn't effect how I was going to use it.  If someday I have to, then I'll tackle it and post an update.
